bin/oneoff/populate_studio_persons

```diff
@@ -0,0 +1,25 @@
+#!/usr/bin/env ruby
+
+# This script creates a studio_people entry (if one does not already exist) for
+# every teacher.
+
+require_relative '../../dashboard/config/environment'
+
+TIME_NOW = DateTime.now.freeze
+
+User.with_deleted.where(user_type: User::TYPE_TEACHER).each do |teacher|
+  # If the teacher already has an associated StudioPerson, skip them.
+  next if teacher.studio_person_id
+
+  # The teacher does not have an associated StudioPerson, so create one,
+  # updating the User to point to the StudioPerson.
+  studio_person = StudioPerson.create(
+    created_at: TIME_NOW,
+    updated_at: TIME_NOW
+  )
+  if teacher.email.blank?
+    teacher.update(studio_person_id: studio_person.id)
```

Comment:
Is there a reason we only set teacher.studio_person_id if email is blank? I would've thought we would do it all the time.